src/main/java/seedu/duke/parsermanager/ParserManager.java

```diff
@@ -98,9 +100,10 @@ public Parser parseCommand(String input) throws DukeParseException, ExistingPair
             if (isClient) {
                 return new ParseCheckClient(commandDetail, clientList);
             } else if (isProperty) {
-                return new ParseCheckProperty(commandDetail, propertyList);
+                return new ParseCheckProperty(commandDetail);
             } else {
-                throw new UndefinedSubCommandTypeException(MESSAGE_MISSING_SUB_COMMAND_TYPE);
+                throw new UndefinedSubCommandTypeException(MESSAGE_CHECK_CLIENT_WRONG_FORMAT
+                        + MESSAGE_CHECK_PROPERTY_WRONG_FORMAT);
```

Comment:
You can consider changing this into a line of variable and pass the parameter inside a as 1 variable.